### PR TITLE
[fix #6201] Update home page Firefox banner

### LINKED ISSUE #6201: [9/25] FFYR - Update homepage banner
This work supports the Fight for Your Rights campaign
 
Success Criteria:
- [ ] Banner live on 9/25

Tasks:
- [x] UI/design of Banner - Jennifer
- [x] Copy for Banner - Natalie/Tim
- [x] Team QA - Design
- [ ] Code Banner - TBD
- [ ] Team QA of Banner on demo 
- [ ] Stakeholder QA of Banner on demo

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Updates the home page Firefox banner for the "Fight for Your Rights" campaign direction.

Also removes Monitor card switch since that's no longer needed

## Issue / Bugzilla link
#6201 

## Testing
The usual: Check for responsiveness and any obvious code bungles.

### CODE REVIEW

**Comment by michaelmccombie** [comment]
Small bug the download button floats to the left and covers the logo on mobile.
image

File: media/css/mozorg/home/home-2018.scss [hunk 1]
```diff
@@ -77,29 +74,37 @@ $callout-blue: #0060df;
...
 
 .download-firefox-primary-cta {
-    background-color: $callout-blue;
+    background-color: #69cafd;
     position: relative;
+    text-align: center;
     z-index: 2;
+
+    .primary-title {
+        font-weight: bold;
```
Comment (alexgibson) [inline media/css/mozorg/home/home-2018.scss]: Should this defintely be bold? The wordmarks are usually regular font weight

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 2]
```diff
@@ -417,7 +403,7 @@ <h2 class="mzp-c-billboard-title">{{ _('Open source. <br>Open minds.') }}</h2>
       <div class="secondary-content">
         <h2 class="secondary-title">{{ _('Supporting a healthy internet is easy.') }}</h2>
         <p class="tagline">{{ _('Get started by browsing with Firefox powered by Mozilla.') }}</p>
-        {{ download_firefox(dom_id='download-secondary', download_location='secondary cta') }}
+        {{ download_firefox(dom_id='download-secondary', download_location='secondary cta', button_color="mzp-c-button mzp-t-download") }}
```
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/mozorg/templates/mozorg/home/home-en.html]: Oops, dev artifact, forgot to clean it up.

File: bedrock/mozorg/templates/mozorg/home/home-en.html [hunk 3]
```diff
@@ -126,18 +124,6 @@ <h3 class="primary-title-sub">{{ _('Fast for good.') }}</h3>
...
             link_url='https://monitor.firefox.com/?utm_source=www.mozilla.org&utm_medium=referral&utm_campaign=homepage&utm_content=card'
         )}}
-      {% else %}
-        {{ card_block(
-            class='mzp-c-card-large',
-            tag_label=_('Branding'),
-            title=_('Evolving the Firefox brand'),
-            ga_title='Evolving the Firefox brand',
-            desc=_('Say “Firefox” and most people think of a web browser on their laptop or phone. There’s more to the story, and our branding needs to evolve.'),
-            image_url='home/2018/cards/design-systems.png',
```
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/mozorg/templates/mozorg/home/home-en.html]: done!

**Pull request merged**